Src/FluentAssertions/Equivalency/AssertionRuleEquivalencyStepAdapter.cs

```diff
@@ -14,7 +14,7 @@ public AssertionRuleEquivalencyStepAdapter(IAssertionRule assertionRule)
 
         public bool CanHandle(IEquivalencyValidationContext context, IEquivalencyAssertionOptions config)
         {
-            return !context.IsRoot;
+            return true;
```

Comment:
❓ I'm not sure what the implications of this change are?

Reply:
That's the real fix. I honestly don't know (or remember) why that condition existed in the first place.